Updated database from PRs 6956, 7058 and 7062

N-thony opened the pull request:
Partially fixes #6185, #6989

@lloyddewit this PR contains the updated database as suggested on PRs #6956, #7058, and #7062 due to the conflicts. Please take a look. Thanks

lloyddewit commented on instat/translations/translationsManuallyEntered.csv:
```diff
@@ -28,3 +28,4 @@ Reset to Default Layout,en,Reset to Default Layout
 View Interactive Plot,en,View Interactive Plot
 Data Frame Metadata,en,Data Frame Metadata
 Script Window,en,Script Window
+Count,en,Count
```
Should this be ' Count' (i.e. preceded by space)? See PR #7062

N-thony commented:
@lloyddewit I have updated the database here rather than opening another PR, and this is because of the small bug in the input station to remembering the state on reopening as related in #6989. Please take a look. Thanks.

lloyddewit commented on instat/translations/translateIgnore.txt:
```diff
@@ -78,9 +78,21 @@ dlgSplitText_ucrInputPattern%
+dlgClimaticBoxPlot_ucrInputWithinYear_cboInput%
+
+# In the 'InventoryPlot' dialog, the selected 'Facet' should not be translated
+# Partially fixes issue, related to PR #7070
```
@N-thony which issue does it fix? Please state issue number. If there is no related issue, then just state `# Related to PR #7070`

lloyddewit commented:
> @lloyddewit I have updated the database here rather than opening another PR, and this is because of the small bug in the input station to remembering the state on reopening as related in #6989. Please take a look. Thanks.

@N-thony the database is a binary file so there is no 'diff' available, and I cannot see what you've changed. If you change the database, then please state exactly which rows you changed (and preferably include a screenshot of the changed rows), thanks

N-thony commented:
> > @lloyddewit I have updated the database here rather than opening another PR, and this is because of the small bug in the input station to remembering the state on reopening as related in #6989. Please take a look. Thanks.
> 
> @N-thony the database is a binary file so there is no 'diff' available, and I cannot see what you've changed. If you change the database, then please state exactly which rows you changed (and preferably include a screenshot of the changed rows), thanks
I have changed `dlgInventoryPlot_ucrInputFacetBy` and `dlgInventoryPlot_ucrInputFacetBy_cboInput` `id_text ` to `DoNotTranslate`
![Uploading image.png…]()

lloyddewit commented:
@N-thony Thank you for the clarification.
In `translateIgnore.txt` you added the following lines:
```
# In the 'Boxplot' dialog, the selected 'Facet' either for Station, Year or Within Year should not be translated
# Fixes issue #6185, item a), related to PR #7058
dlgClimaticBoxPlot_ucrInputStation%
dlgClimaticBoxPlot_ucrInputYear%
dlgClimaticBoxPlot_ucrInputWithinYear%
dlgClimaticBoxPlot_ucrInputWithinYear_cboInput%
```
However the database contains the following:

![image](https://user-images.githubusercontent.com/57253949/148981146-c7697cb4-4e3c-42df-b5ab-f652fb6ce972.png)
Should these rows be `DoNotTranslate`?

lloyddewit commented on instat/translations/translateIgnore.txt:
```diff
@@ -78,9 +78,21 @@ dlgSplitText_ucrInputPattern%
+dlgClimaticBoxPlot_ucrInputStation%
+dlgClimaticBoxPlot_ucrInputYear%
+dlgClimaticBoxPlot_ucrInputWithinYear%
+dlgClimaticBoxPlot_ucrInputWithinYear_cboInput%
```
This line is not needed because it's covered by the line above ('%' is a wildcard).
```suggestion
```

lloyddewit commented on instat/translations/translateIgnore.txt:
```diff
@@ -78,9 +78,21 @@ dlgSplitText_ucrInputPattern%
+# In the 'InventoryPlot' dialog, the selected 'Facet' should not be translated
+# Partially fixes issue #6989, related to PR #7070
+dlgInventoryPlot_ucrInputFacetBy%
+dlgInventoryPlot_ucrInputFacetBy_cboInput%
```
This line is not needed because it's covered by the line above ('%' is a wildcard).
```suggestion
```

lloyddewit commented:
@shadrackkibet Please could you test whether this PR fully resolves issue #6185? If yes, then please could you approve and merge this PR? thanks

shadrackkibet commented:
The bug reported still exists.

![image](https://user-images.githubusercontent.com/26301973/149149340-faf97768-1e31-406f-9ddc-a4d679519bdc.png)

N-thony commented:
> The bug reported still exists.
> 
> ![image](https://user-images.githubusercontent.com/26301973/149149340-faf97768-1e31-406f-9ddc-a4d679519bdc.png)

@shadrackkibet thanks for your review. I think this PR concerns item a) on issue #6185, it partially fixes that issue and another issue of control not remembering the state on reopening due to the translation. I may fix that bug in another PR.

N-thony commented:
> > The bug reported still exists.
> > ![image](https://user-images.githubusercontent.com/26301973/149149340-faf97768-1e31-406f-9ddc-a4d679519bdc.png)
> 
> @shadrackkibet thanks for your review. I think this PR concerns item a) on issue #6185, it partially fixes that issue, and other issues of control not remembering the state on reopening due to the translation. I may fix that bug in another PR.

I got an issue with my local branch, I'm cloning again the repo and then I will take a look of the bug. Thanks

shadrackkibet reviewed (approved):
Great. I will merge these changes for now while you consider fixing the other bugs in a separate PR.
